library/src/main/scala/scala/scalajs/js/Tuple.scala

```diff
@@ -0,0 +1,697 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js API               **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-lang.org/     **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+/**
+ * All doc-comments marked as "MDN" are by Mozilla Contributors,
+ * distributed under the Creative Commons Attribution-ShareAlike license from
+ * https://developer.mozilla.org/en-US/docs/Web/Reference/API
+ */
+package scala.scalajs.js
+
+import scala.scalajs.js.annotation.JSName
+import scala.language.implicitConversions
+
+/**
+ *  Tuple "view" of a JavaScript array.
```

Comment:
This doc-comment is unfortunately useless as a ScalaDoc, since it is not attached to any class.

I suggest to attach a detailed ScalaDoc to `js.Tuple2`, and have a very simple "see also" kind of link on the other tuple types.